README.md

```diff
@@ -85,11 +85,11 @@ If you want to load a different flavor, like Hey Willamette, just replace the fi
 NOTE: Flavors can load data from any number of Shareabouts datasets, provided you have a URL and key for each dataset you'd like to load. Dataset URLs and keys are set in your `.env` file, as follows:
 
 ```
-<DATASET-ID>_SITE_URL=https://path/to/dataset/
-<DATASET-ID>_DATASET_KEY=dataset_key
+DATASET-ID_SITE_URL=https://path/to/dataset/
+DATASET-ID_DATASET_KEY=dataset_key
```

Comment:
I can't quite remember why we decided to remove the angle brackets, although I do remember having a discussion about it at some point. Do you think it's better to leave them in?